src/templates/base.html

```diff
@@ -145,6 +160,10 @@
 
     USER_IS_SUPERUSER = "{{ user.is_superuser }}" === "True"
 
+    USERNAME = "{{ user.username }}"
```

Comment:
Probably need to re-visit this as I did `SITE.state.user` since `CHAHUB` was already a declared variable... I think semantically this makes sense too?